Service/Form/Submission/FormSubmissionService.php

```diff
@@ -91,4 +91,24 @@ public function submit(FormSubmissionRequest $submitRequest): array
 
         return ['submission_id' => $formSubmission->getId()];
     }
+
+    public function removeExpiredSubmissions(): int
+    {
+        $removedCount = 0;
+
+        foreach ($this->repository->findAll() as $submission) {
+            $deadline = $submission->getDeadlineDate();
+            if (!\is_string($deadline)) {
+                continue;
+            }
+
+            $expireTimestamp = \strtotime($deadline);
```

Comment:
In the get getDeadlineDate function?

Reply:
Deadline should not be a string but DB  in remark, so this conversion should not be needed.